From e82bb38adfc9edfacfe7118592eb4b4357cc687b Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Mon, 27 Jan 2025 17:49:39 +0100
Subject: [PATCH] libotutil: Remove redundant import of prctl.h

sys/prctl.h already provides prctl.h and adding both imports fails on
musl libc:

	/usr/include/sys/prctl.h:88:8: error: redefinition of 'struct prctl_mm_map'
	   88 | struct prctl_mm_map {
	      |        ^~~~~~~~~~~~
	In file included from src/libotutil/ot-unix-utils.c:33:
	/usr/include/linux/prctl.h:134:8: note: originally defined here
	  134 | struct prctl_mm_map {
	      |        ^~~~~~~~~~~~
---
 src/libotutil/ot-unix-utils.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libotutil/ot-unix-utils.c b/src/libotutil/ot-unix-utils.c
index 551cc8789d..eee3a44f57 100644
--- a/src/libotutil/ot-unix-utils.c
+++ b/src/libotutil/ot-unix-utils.c
@@ -30,7 +30,6 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <linux/capability.h>
-#include <linux/prctl.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/prctl.h>
